package.json

```diff
@@ -6,7 +6,8 @@
     },
     "optionalDependencies": {
         "diskusage": "^1.1.3",
-        "greenlock": "^2.8.8",
+        "greenlock": "^4.0.4",
```

Comment:
This will make #610 obsolete.

Reply:
dependabot should detect and auto close that